cmd/gateway-router.go

```diff
@@ -36,8 +36,8 @@ type GatewayLayer interface {
 	GetBucketPolicies(string) (policy.BucketAccessPolicy, error)
 	DeleteBucketPolicies(string) error
 	AnonListObjects(bucket, prefix, marker, delimiter string, maxKeys int) (result ListObjectsInfo, err error)
-	AnonListObjectsV2(bucket, prefix, continuationToken string, fetchOwner bool, delimiter string, maxKeys int) (result ListObjectsV2Info, err error)
-	ListObjectsV2(bucket, prefix, continuationToken string, fetchOwner bool, delimiter string, maxKeys int) (result ListObjectsV2Info, err error)
+	AnonListObjectsV2(bucket, prefix, continuationToken, delimiter string, maxKeys int, fetchOwner bool, startAfter string) (result ListObjectsV2Info, err error)
```

Comment:
we could keep `startAfter` with other `string` type arguments.

Suggestion:
```go
AnonListObjectsV2(bucket, prefix, continuationToken, delimiter, startAfter string, maxKeys int, fetchOwner bool) (result ListObjectsV2Info, err error)
```

Reply:
It can, but I wanted to keep less used params at end.  Let me know whether we still need to move it to string argument group.

Reply:
@balamurugana OK, you can leave arguments as is.